integrationtest/performance/src/test/java/org/hibernate/search/test/performance/scenario/TestScenario.java

```diff
@@ -90,7 +90,6 @@ public Properties getHibernateProperties() {
 		Properties properties = new Properties();
 		properties.setProperty( "hibernate.hbm2ddl.auto", "create" );
 		properties.setProperty( "hibernate.search.default.lucene_version", "LUCENE_CURRENT" );
-		properties.setProperty( "hibernate.search.index_uninverting_allowed", "false" );
```

Comment:
We have a similar setup `org.hibernate.search.test.query.sorting.SortOnFieldsFromCustomBridgeTest.configure(Map<String, Object>)`. Did you leave it on purpose?